client/landing/stepper/declarative-flow/internals/steps-repository/woo-confirm/index.tsx

```diff
@@ -119,24 +102,18 @@ const WooConfirm: Step = function WooCommerceConfirm( { navigation } ) {
 
 	// Filter the Woop transferring blockers.
 	const transferringBlockers = eligibilityHolds?.filter(
-		( hold: TransferEligibilityError ) => ! TRANSFERRING_NOT_BLOCKERS.includes( hold.code )
+		( hold: TransferEligibilityHold ) => ! TRANSFERRING_NOT_BLOCKERS.includes( hold )
 	);
 
 	const isTransferStuck = latestAtomicTransfer?.is_stuck || false;
 	const isBlockByTransferStatus = latestAtomicTransferError || false;
 
 	// Add blocked-transfer-hold when something is wrong in the transfer status.
 	if (
-		! transferringBlockers?.includes( {
-			code: eligibilityHoldsConstants.BLOCKED_ATOMIC_TRANSFER,
-			message: '',
-		} ) &&
+		! transferringBlockers?.includes( 'BLOCKED_ATOMIC_TRANSFER' ) &&
 		( isBlockByTransferStatus || isTransferStuck )
 	) {
-		transferringBlockers?.push( {
-			code: eligibilityHoldsConstants.BLOCKED_ATOMIC_TRANSFER,
-			message: '',
-		} );
+		transferringBlockers?.push( 'BLOCKED_ATOMIC_TRANSFER' );
```

Comment:
I feel like this is a logic error -- when `transferringBlockers` is null (e.g. when there are no errors at all), then this check will never add this error. I don't know if this is intentional, and the logic in this file is complicated enough I didn't want to mess with it too much, beyond fixing the obviously wrong type mismatch.

Reply:
Fair enough. I've double-checked that we're keeping the logic as it was, which is what we should be after in this PR essentially.